mmcv/transforms/utils.py

```diff
@@ -79,10 +83,43 @@ def __get__(self, obj, cls):
         return self
 
 
+def prohibit_cache_randomness(cls):
```

Comment:
1. would avoid_cache_randomness more easy to understand?
2. should we consider the case like below:
```python

@prohibit_cache_randomness
class A(BaseTransforms):
    pass

class B(A):
    @cache_randomness
    def random_func(self)
```

Reply:
For question 2, I am wondering if the subclass should inherit the attribute "avoid_cache_randomness" or not?

Reply:
If we want to allow case 2, I think then class B should have different avoid_cache_randomness value from A

Reply:
Ok, we can do that.